extensions/blocks/slideshow/style.scss

```
@@ -236,3 +236,39 @@
 		}
 	}
 }
+
+@media only email {
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_container {
+		overflow: visible;
+		opacity: 1;
+		width: auto;
+		height: auto;
+	}
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_container ul.wp-block-jetpack-slideshow_swiper-wrapper,
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_slide figure {
+		display: block;
+		margin-bottom: 12px;
+	}
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_container ul.wp-block-jetpack-slideshow_swiper-wrapper,
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_slide {
+		list-style: none;
+		margin-left: auto;
+		margin-right: auto;
+	}
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_slide {
+		display: inline-block;
+		width: 42%;
+		margin-left: 2% !important;
+		margin-right: 2% !important;
+		height: auto;
+		vertical-align: top;
+	}
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_caption.gallery-caption {
+		position: relative;
+		right: auto;
+		bottom: auto;
+		background-color: transparent;
+		color: #2c3338;
```

Maybe it's better to inherit the default text color?
```suggestion
		color: inherit;
```

---

Good call!